lms/static/scripts/frontend_apps/components/test/FilePickerApp-test.js

```diff
@@ -100,35 +100,35 @@ describe('FilePickerApp', () => {
 
   it('renders buttons to choose assignment source', () => {
     const wrapper = renderFilePicker();
-    assert.equal(wrapper.find('Button').length, 2);
+    assert.equal(wrapper.find('LabeledButton').length, 2);
   });
 
   it('renders Canvas file picker button if Canvas file picker enabled', () => {
     fakeConfig.filePicker.canvas.enabled = true;
     const wrapper = renderFilePicker();
-    assert.isTrue(wrapper.exists('Button[label="Select PDF from Canvas"]'));
+    assert.isTrue(wrapper.exists('LabeledButton[data-testid="pdf-button"]'));
```

Comment:
👍 - Changing label matching to `data-testid` seems good here. We'll need to do this more widely when we add support for other languages which might happen later this year.